tests/out/quad-vert.msl

```diff
@@ -35,7 +37,7 @@ struct main2Output {
     metal::float2 member [[user(loc0), center_perspective]];
     metal::float4 gl_Position1 [[position]];
     float gl_PointSize1 [[point_size]];
-    type6 gl_ClipDistance1 [[clip_distance]];
+    float gl_ClipDistance1 [[clip_distance]] [1];
```

Comment:
this seems wrong. Why is it not written as `float gl_ClipDistance1[1] [[clip_distance]];`?

Reply:
Nope, the compiler doesn't like that:
```
testfrag.metal:40:34: error: 'clip_distance' attribute cannot be applied to types
    float gl_ClipDistance1 [1] [[clip_distance]];
                                 ^
testfrag.metal:42:8: error: invalid return type 'main2Output' for vertex function
vertex main2Output main2(
       ^
testfrag.metal:40:5: note: field of illegal type 'float [1]' declared here
    float gl_ClipDistance1 [1] [[clip_distance]];
    ^
2 errors generated.
```